examples/metrics/kafka-connect-metrics.yaml

```diff
@@ -13,7 +13,7 @@ spec:
     valueFrom:
       configMapKeyRef:
         name: connect-metrics
-        key: metrics-config.yml
+        key: metrics-config.json
```

Comment:
This is actually YAML ... so this should stay as it was ... no cheating with _replace all_ ;-)